MapboxNavigation/CPFavoritesList.swift

```diff
@@ -0,0 +1,58 @@
+#if canImport(CarPlay)
+import CarPlay
+#endif
+
+public enum CPFavoritesList {
```

Comment:
The `CP` prefix is reserved for the CarPlay framework itself. “CarPlay” should be spelled out.